ksql-engine/src/test/java/io/confluent/ksql/statement/PropertyOverrideInjectorTest.java

```diff
@@ -79,15 +84,13 @@ public void shouldFailOnInvalidSetPropertyValue() {
     expectedException.expectMessage("Invalid value invalid");
 
     // When:
-    CustomValidators.SET_PROPERTY.validate(
-        ConfiguredStatement.of(
+    new PropertyOverrideInjector().inject(
+      ConfiguredStatement.of(
         PreparedStatement.of(
              "SET '" + ConsumerConfig.AUTO_OFFSET_RESET_CONFIG + "' = 'invalid';",
             new SetProperty(Optional.empty(), ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "invalid")),
             new HashMap<>(),
-            engine.getKsqlConfig()),
-        engine.getEngine(),
-        engine.getServiceContext()
+            ksqlConfig)
```

Comment:
nit: the spacing here is weird since `new HashMap<>()` and `ksqlConfig` are parameters to the `ConfiguredStatement` call, not the `PreparedStatement` call. This weirdness also appears in multiple other tests in this file.

Reply:
nit: the spacing here is still funky. (The weirdness appears in multiple places throughout the file.)

Reply:
Huh... I'll comb through and see what's going on. My IntelliJ settings might have gone wacky.